### DISCUSSION TITLE
增加顶部模块菜单

### DISCUSSION THREAD

**Discussion by 312977** [vbenjs/vue-vben-admin#3122] [General]
### Subject of the featur

增加顶部增加模块菜单
### Problem

当项目菜单太多的时候，会出现左侧太长，希望顶部能加模块菜单，这样，每个模块切换后 左侧菜单不同

**Comment by WitMiao** [comment]
<img width="332" alt="image" src="https://user-images.githubusercontent.com/38201220/229712578-bce03e54-743a-4eda-93d4-3e97f02d94f1.png">
<img width="303" alt="image" src="https://user-images.githubusercontent.com/38201220/229712743-39d479eb-a242-4825-8f51-e4cb1cda2013.png">
试试这个

**Comment by 312977** [comment]
![image](https://user-images.githubusercontent.com/3642374/230615403-d1deb814-3572-4f81-8b9f-1206fa201057.png)
![image](https://user-images.githubusercontent.com/3642374/230615484-aa22c6ca-feb0-4ad1-a529-f3fe3e9a5e29.png)
我说的是这种

**Comment by one-god** [comment]
> 这种

人家都教你怎么开启了

**Comment by xiaomoumou** [comment]
![image](https://user-images.githubusercontent.com/38878071/231925021-f0dd1b25-d24d-4bbb-82fb-cdfbb27d36fc.png)

选【导航栏模式】第二个模式，然后开启【分割菜单】选项。
> ![image](https://user-images.githubusercontent.com/3642374/230615403-d1deb814-3572-4f81-8b9f-1206fa201057.png) ![image](https://user-images.githubusercontent.com/3642374/230615484-aa22c6ca-feb0-4ad1-a529-f3fe3e9a5e29.png) 我说的是这种

**Comment by sunfeng90** [comment]
如何默认就是顶部菜单模式？不需要手动设置的，我们只需要顶部菜单模式，不需要左侧的菜单模式

**Reply by AidenYan97** [reply]
官方文档上有
![image](https://github.com/vbenjs/vue-vben-admin/assets/51433157/3c633f50-4cb9-4925-a870-d246a4339734)

**Comment by AidenYan97** [comment]
[vben-admin-thin-next](https://vben.vvbin.cn/thin/next/) - Simplified Chinese site
这个版本这样操作好像有问题，设置之后点击顶部菜单（只有一级的没有子级）不显示新点击的菜单的页面

**Reply by likui628** [reply]
这个版本不维护了
